brewtils/rest/easy_client.py

```diff
@@ -39,7 +39,7 @@ def get_easy_client(**kwargs):
         **kwargs: Options for configuring the EasyClient
 
     Returns:
-        :obj:`brewtils.rest.easy_client.EasyClient`: The configured client
+        brewtils.rest.easy_client.EasyClient: The configured client
```

Comment:
It wasn't immediately obvious to me what the whole `:obj` thing was trying to do.  Just having the text here results in the proper linkages within the documentation.

Reply:
It's Sphinx domain syntax. It should have been `:py:obj:\`brewtils.rest.easy_client.EasyClient\``. At any rate, the napoleon pre-processor evidently makes that syntax obsolete.

Reply:
Good to know. Thanks!